libs/auth/src/pages/cft-login/return.ts

```diff
@@ -0,0 +1,3 @@
+// Re-export from callback folder to maintain /cft-login/return URL
+// This URL is configured on the CFT IDAM side and cannot be changed
+export { GET } from "../callback/cft.js";
```

Comment:
This is very weird. It is re-exporting a method from another class in a different folder. Can me remove this and export directly from the other class